.github/workflows/ci-backend-cql.yml

```diff
@@ -70,6 +70,8 @@ jobs:
             args: "-Pcassandra3-murmur -Dtest=\"**/graphdb/cql/*\""
           - module: cql
             args: "-Pcassandra3-murmur -Dtest=\"**/hadoop/*\""
+          - module: cql
+            args: "-Pcassandra3-murmur -Dtest=\"**/core/cql/*\""
```

Comment:
Can you add
```
          - module: cql
            args: "-Pcassandra3-byteordered -Dtest=\"**/core/cql/*\""
```
so that we test against byteordered partitioner as well?